[17] Socialie/Socialie/Views/Settings View/SettingsView.swift

```diff
@@ -55,14 +56,14 @@ private extension SettingsView {
 	private var haptics: some View {
 		Toggle(
 			"Enable Haptics",
-			isOn: $viewModel.isHapticsEnabled
+			isOn: $isHapticsEnabled
 		)
 		.tint(.accentColor)
```

Comment:
Got code for haptics but haptic functionality is not implemented yet in the app 😅

Reply:
I agree, at the moment temporarily removed the switcher for haptics. Would add it back later once I figured out how to implement haptics appropriately throughout the app